### PR TITLE
Fix english mistake

### PR DESCRIPTION
**Pull request by marie-bnl**
I think "searching for" is a mistake and it should be either "searching" or "looking for". So I changed it to "looking for".

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,7 +15,7 @@ <h1 class="mb-3" href="{{ site.baseurl }}{% link index.html %}">
                 configured in <a href="https://www.haskell.org/">Haskell</a>.</p>
             <p class="lead mb-3">
                 In a normal WM, you spend half
-                your time aligning and searching for windows. XMonad
+                your time aligning and looking for windows. XMonad
```
Comment (geekosaur) [inline index.html:18]: As a native English speaker, "searching for" seems fine to me.

Edit: I should probably specify: midwestern U.S.

**Comment by marie-bnl** [comment]
You must be right then, I'm not a native speaker. My bad ^^

**Comment by geekosaur** [comment]
Well, I can't speak for what a British English speaker would say.

**Comment by marie-bnl** [comment]
I think my English tends to be more Britain but I don't know if it differs on this. Anyway I guess most people actually speak a US English.

**Comment by liskin** [comment]
A non-native UK English speaker wielding the online Cambridge English dictionary agrees with @geekosaur that "searching for" is correct. That being said, "looking for" sounds a bit better. Also, the dictionary says there is a difference in meaning:

look = to try to find something or someone
search = to look somewhere carefully in order to find something

(But I promised myself that I'll try to reduce the bikeshedding, so… let's just leave it as it is, and if/when anyone comes and improves/rewrites the whole section/page, then they can reword this.)

**Comment by marie-bnl** [comment]
Okay interesting! And yes of course that's understandable

**Pull request closed without merging**